{{cookiecutter.repo_name}}/src/pages/[...slug].jsx

```diff
@@ -3,35 +3,34 @@ import { getPageBySlug } from 'util/api';
 import StreamField from 'components/streamfield/StreamField';
 
 const ContentPage = ({ data }) => {
-    return <>{data && <StreamField stream={data?.body} />}</>;
+  return <>{data && <StreamField stream={data?.body} />}</>;
 };
 
 ContentPage.propTypes = {
-    data: PropTypes.object,
+  data: PropTypes.object,
 };
 
 ContentPage.defaultProps = {
-    data: {},
+  data: null,
```

Comment:
Hm it looks like your formatter isn't using the editorconfig file (assuming there is one) for this change and several other indent changes in this file

Reply:
Otherwise this change looks god!

Reply:
@brianedelman looks like the `.editorconfig` file expects two-space indents rather than 4. How do we normally indent jsx files?

```
[*]
end_of_line = lf
charset = utf-8
trim_trailing_whitespace = true
insert_final_newline = true
indent_style = space
indent_size = 4

[*.{js,jsx}]
indent_style = space
indent_size = 2
```

Reply:
Yeah we talked about this online the other day but I think this is fine and we should do a pass where we fix spacing in a separate PR as it seems to be wrong in these files